src/createMollieClient.ts

```diff
@@ -46,7 +47,8 @@ function createHttpClient(options: MollieOptions): AxiosInstance {
   axiosOptions.headers['Accept-Encoding'] = 'gzip';
   axiosOptions.headers['Content-Type'] = 'application/json';
 
-  var customVersionStrings = options.versionStrings;
+  let customVersionStrings = options.versionStrings;
+
   if (undefined == customVersionStrings) {
```

Comment:
we use Yoda? and shoudn't this be `===`?

Reply:
That depends on the behaviour we desire should someone ever pass `null` as `versionStrings`.

`==` means it will be treated the same as not passing any version strings. `===` means it will throw an exception, which one could argue is to be expected.